### PR TITLE
French translations

### PR DESCRIPTION
**Pull request by ebeauchamps**

### CODE REVIEW

File: resources/lang/fr/messages.php [hunk 1]
```diff
@@ -67,12 +70,12 @@
...
     'entry_origin_instructions' => 'La nouvelle traduction héritera des valeurs qu’a l’entrée dans le site sélectionné.',
     'expect_root_instructions' => 'Considérer la première page de l’arborescence comme une page "racine" ou "d’accueil".',
-    'field_conditions_always_save_instructions' => 'Toujours enregistrer la valeur du champ, quelle que soit la façon dont les conditions du champ sont évaluées.',
     'field_conditions_instructions' => 'Quand afficher ou masquer ce champ.',
     'field_desynced_from_origin' => 'Désynchronisé de l’origine. Cliquez pour synchroniser et revenir à la valeur d’origine.',
     'field_synced_with_origin' => 'Synchronisé avec l’origine. Cliquez ou modifiez le champ pour désynchroniser.',
     'field_validation_advanced_instructions' => 'Ajouter des règles de validation plus avancées à ce champ.',
     'field_validation_required_instructions' => 'Permet de contrôler si ce champ est obligatoire ou non.',
+    'fields_always_save_instructions' => 'Toujours enregistrer la valeur du champ, quelle que soit la façon dont les conditions du champ sont évaluées.',
```
Comment (jasonvarga) [inline resources/lang/fr/messages.php:78]: Can you merge 3.4 back into this branch? It looks like an issue. The `fields_always_save_instructions` key was recently renamed to `field_conditions_always_save_instructions`. Here you are reverting that.

Comment (ebeauchamps) [reply resources/lang/fr/messages.php:78]: I'll try to do that

Comment (jasonvarga) [reply resources/lang/fr/messages.php]: I see that maybe you're having trouble with Git here. Sorry I didn't mean to complicate it for you. I don't mind fixing it.

Comment (ebeauchamps) [reply resources/lang/fr/messages.php]: yeah, I have never been very "git-savvy"...sorry about that. Is it OK now?

**Review by jasonvarga — APPROVED** [review]
Looks good to me. Merci!

**Comment by ebeauchamps** [comment]
@jasonvarga Sorry to bother you, but I am running 3.4.0 and right now I have a problem with the translation related to our discussion just above, as ```'fields_always_save_instructions'``` was translated but right now ```field_conditions_always_save_instructions``` isn't.  Was the later introduced in 3.4.1?


Capture d’écran 2023-02-02 à 21 44 48

**Comment by jasonvarga** [comment]
That string was renamed in #7417, which has not been released yet.

**Pull request merged**